salt/state.py

```diff
@@ -1648,6 +1649,8 @@ def call(self, low, chunks=None, running=None, retries=1):
         else:
             ret = {'result': False, 'name': low['name'], 'changes': {}}
 
+	self.state_con['runas'] = low.get('runas', None)
```

Comment:
Woops, it was not, i really need to trash my vimrc and start over, because it keeps disabling expandtab for some reason.